pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = --ds=osf_tests.settings --tb=short --reuse-db --allow-hosts=127.0.0.1
+addopts = --ds=osf_tests.settings --tb=short --reuse-db --allow-hosts=127.0.0.1,192.168.168.167
```

Comment:
Why is this necessary, local testing?

Reply:
yes, it would be nice if we could just leave this in